### ISSUE TITLE
Fixed statement should follow csharp_space_after_keywords_in_control_flow_statements

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn#37031]
The formatter does not account for `csharp_space_after_keywords_in_control_flow_statements` when formatting a `fixed` statement, but should.

Set `csharp_space_after_keywords_in_control_flow_statements` to `false` and then format the following:

```csharp
class SomeType {
  unsafe void Method() {
    fixed (byte* x = new byte[3]) {
    }
  }
}
```

### Expected result

The space is removed:

```csharp
class SomeType {
  unsafe void Method() {
    fixed(byte* x = new byte[3]) {
    }
  }
}
```

### Actual result

The spacing is not changed.

**Comment by nbalakin** [comment]
Going to fix the issue.

**Comment by nbalakin** [comment]
Ok, this seems to be an easy one. 🙂

**Comment by Youssef1313** [comment]
@sharwell This is already fixed in #37827 and can be closed now.

**Comment by sharwell** [comment]
Thanks @nbalakin and @Youssef1313 !
